tests/local_client_server_tcp_e2e/local_client_server_tcp_e2e.c

```diff
@@ -26,6 +26,23 @@ static TEST_MUTEX_HANDLE g_dllByDll;
 
 #define TEST_TIMEOUT 30 // seconds
 
+static int generate_port_number()
+{
+	int port_number;
+	const char* comp_time = __TIME__;
```

Comment:
>__TIME__ [](start = 25, length = 8)

Let's just use rand() as Jelani said. Adding extra salt cannot be quantified, since the __TIME__ will be the same in some cases anyhow. __TIME__ is expanded at preprocessor time anyhow. Are you sure the translation unit even gets compiled more than one time?

Taking all of this into account, I'd say let's go with the simple rand for now. If you want bulletproof I think you need another solution.